src/test/java/org/broadinstitute/hellbender/utils/read/markduplicates/ReadsKeyUnitTest.java

```diff
@@ -0,0 +1,79 @@
+package org.broadinstitute.hellbender.utils.read.markduplicates;
+
+import org.testng.annotations.Test;
+
+import java.util.Random;
+
+public class ReadsKeyUnitTest {
+
+    @Test //(enabled = false)
+    // This test makes the argument that using a StringBuilder in ReadsKey is faster than an equivalent call to String.format()
+    /* an example of the results:
+        With Format:	13909867289
+        With Builder:	1130077051
+        With String +:	1155343164
+        With Joiner:	2013227121
+     */
+    public void testKeyForFragment() throws Exception {
```

Comment:
Rename this test case so that it's clear it's a performance test only (include the word "performance" in the name)